buildSrc/src/main/resources/gocd-docker-server/docker-entrypoint.sh

```diff
@@ -82,12 +64,9 @@ if [ "$1" = '/go-server/server.sh' ]; then
         yell "Ignoring $file, it is either not a file or is not executable"
       fi
     done
-
-    try exec /usr/local/sbin/tini -- /usr/local/sbin/gosu go "$0" "$@"
-  fi
 fi
 
 # these 3 vars are used by `/go-server/server.sh`, so we export
 export GO_SERVER_SYSTEM_PROPERTIES="${GO_SERVER_SYSTEM_PROPERTIES}${GO_SERVER_SYSTEM_PROPERTIES:+ }-Dgo.console.stdout=true"
 
-try exec "$@"
+try /usr/local/sbin/tini exec "$@"
```

Comment:
we should be executing tini with args. Current usage is incorrect and gives the following error 

```╰─$ docker run -it docker-gocd-server:v19.5.0-9192                                                                                                                                        [19/06/5| 1:55PM]
/docker-entrypoint.sh: Creating directories and symlinks to hold GoCD configuration, data, and logs
$ mkdir -v /godata/artifacts
created directory: '/godata/artifacts'
$ ln -sv /godata/artifacts /go-working-dir/artifacts
'/go-working-dir/artifacts' -> '/godata/artifacts'
$ mkdir -v /godata/config
created directory: '/godata/config'
$ ln -sv /godata/config /go-working-dir/config
'/go-working-dir/config' -> '/godata/config'
$ mkdir -v /godata/db
created directory: '/godata/db'
$ ln -sv /godata/db /go-working-dir/db
'/go-working-dir/db' -> '/godata/db'
$ mkdir -v /godata/logs
created directory: '/godata/logs'
$ ln -sv /godata/logs /go-working-dir/logs
'/go-working-dir/logs' -> '/godata/logs'
$ mkdir -v /godata/plugins
created directory: '/godata/plugins'
$ ln -sv /godata/plugins /go-working-dir/plugins
'/go-working-dir/plugins' -> '/godata/plugins'
$ mkdir -v /godata/addons
created directory: '/godata/addons'
$ ln -sv /godata/addons /go-working-dir/addons
'/go-working-dir/addons' -> '/godata/addons'
$ cp -rfv /go-server/config/logback-include.xml /go-working-dir/config/logback-include.xml
'/go-server/config/logback-include.xml' -> '/go-working-dir/config/logback-include.xml'
$ install-gocd-plugins
$ mkdir -p /godata/plugins/external
$ git-clone-config
$ cd /godata/config
/docker-entrypoint.sh: Running custom scripts in /docker-entrypoint.d/ ...
$ /usr/local/sbin/tini exec /go-server/server.sh
[WARN  tini (25)] Tini is not running as PID 1 and isn't registered as a child subreaper.
Zombie processes will not be re-parented to Tini, so zombie reaping won't work.
To fix the problem, use the -s option or set the environment variable TINI_SUBREAPER to register Tini as a child subreaper, or run Tini as PID 1.
[FATAL tini (26)] exec exec failed: No such file or directory
/docker-entrypoint.sh: cannot /usr/local/sbin/tini exec /go-server/server.sh
```


change usage to 

`try exec /usr/local/sbin/tini "$@"`